.github/workflows/build_and_test.yml

```diff
@@ -10,7 +10,11 @@ on:
 jobs:
   build-and-test:
     name: Build and Test Web and Electron Apps
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+      fail-fast: false
```

Comment:
I think it will actually lead to more expense because it will require us to first fully fix one platform before we can discover if there are bugs in other platforms.  Better to get all of the results at once.